modules/mysql/mysql_test.go

```diff
@@ -0,0 +1,223 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	// Import mysql into the scope of this package (required)
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func TestMySQL(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := StartContainer(ctx, "mysql:8")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Clean up the container after the test is complete
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	// perform assertions
+
+	host, _ := container.Host(ctx)
+
+	p, _ := container.MappedPort(ctx, "3306/tcp")
+	port := p.Int()
+
+	connectionString := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?tls=skip-verify",
```

Comment:
In the postgres module we are proposing a method in the container retrieving the connection string, which I consider handy for databases. Do you think we could replicate it for MySQL (and eventual future databases?)

Reply:
sure! I can check it later